portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -89,21 +89,17 @@ private String getStringParameter(HttpServletRequest request, String name, Strin
 
   /** Get number of comments to display from request */ 
   private int getNumCommentsParameter(HttpServletRequest request) {
-    String numCommentsString = request.getParameter(NUM_COMMENTS_PARAMETER);
+    String numCommentsString = request.getStringParameter(request, NUM_COMMENTS_PARAMETER, "");
```

Comment:
I meant simply call `getStringParameter()` in your class here. It would look like:

```
String numCommentsString = getStringParameter(request, NUM_COMMENTS_PARAMETER, "");
```